### DISCUSSION TITLE
Multiple tables in one page

### DISCUSSION THREAD

**Discussion by Harold2017** [QuestPDF/QuestPDF#417] [Q&A] [answered]
Related issue: [#258](https://github.com/QuestPDF/QuestPDF/issues/258)

Hi all,

When i tried to add multiple tables to a single page (container), it thows the exception as follows:

```
dotnet : Unhandled exception. QuestPDF.Drawing.Exceptions.DocumentComposeException: You should not assign multiple child elements to a si
ngle-child container. This may happen when a container variable is used outside of its scope/closure OR the container is used in multiple
 fluent API chains OR the container is used incorrectly in a loop. This exception is thrown to help you detect that some part of the code
 is overriding fragments of the document layout with a new content - essentially destroying existing content.
```

I understand it warns me NOT use same child element type multiple times in a container (context), but how can i add multiple tables in a single page?

I'd appreciate any suggestion.

Thanks.

**Comment by bennetbo** [comment, marked as answer]
You are probably looking for the [Column](https://www.questpdf.com/api-reference/column) element.
Which you can use like this:
```csharp
container.Column(c =>
{
    c.Table(t => // Table 1);
    c.Table(t => // Table 2);
});
```

It looks like you are just getting started with QuestPDF, so its definitly worth to checkout the [Getting Started](https://www.questpdf.com/getting-started) tutorial, which will explain all this.

**Reply by Harold2017** [reply]
Yes, just a newbie for this lib, thanks for your reply!

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
